net/grpc.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package net
+
+import (
+	"context"
+
+	"google.golang.org/grpc"
+)
+
+const (
+	serviceGetDocGraphName  = "/defradb.net.Service/GetDocGraph"
```

Comment:
suggestion: Maybe define a string const for `"defradb.net.Service"` and use it here and below.

Reply:
done